x-pack/plugins/cloud_security_posture/public/pages/findings/findings.tsx

```diff
@@ -39,23 +87,12 @@ export const Findings = () => {
             />
           )}
         />
-        <Route
-          path={findingsNavigation.findings_default.path}
-          render={() => (
-            <TrackApplicationView viewId={findingsNavigation.findings_default.id}>
-              <LatestFindingsContainer dataView={dataViewQuery.data!} />
-            </TrackApplicationView>
-          )}
-        />
-        <Route
-          path={findingsNavigation.findings_by_resource.path}
-          render={() => <FindingsByResourceContainer dataView={dataViewQuery.data!} />}
-        />
-        <Route
-          path={'*'}
-          component={() => <Redirect to={findingsNavigation.findings_default.path} />}
-        />
+
+        <Route path={findingsNavigation.findings_default.path} component={Configurations} />
+        <Route path={findingsNavigation.vulnerabilities.path} component={Vulnerabilities} />
```

Comment:
considering putting the `configurations` and `vulnerabilities` folders inside the `findings` folder, to better represent the UI structure